xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3647,6 +3646,8 @@ void CVideoPlayer::SetSpeed(float speed)
   
   m_newPlaySpeed = speed * DVD_PLAYSPEED_NORMAL;
   SetPlaySpeed(speed * DVD_PLAYSPEED_NORMAL);
+  if (speed == 1)
```

Comment:
you need to compare to old speed or you get resume calls even if nothing changed.